handlers/redirect_handler.go

```diff
@@ -25,14 +25,22 @@ func addCacheHeaders(writer http.ResponseWriter) {
 	writer.Header().Set("Cache-Control", fmt.Sprintf("max-age=%d, public", cacheDuration/time.Second))
 }
 
+func addGAQueryParam(url string, request *http.Request) string {
+	if ga := request.URL.Query().Get("_ga"); ga != "" {
+		return url + "?_ga=" + ga
```

Comment:
If the destination URL already has a query string (which some do), then this will be a bit wonky

Reply:
Ah, thanks. I forgot about this case.